### PR TITLE
Update ruby script

### PR DESCRIPTION
**Pull request by onmyway133**
- Infer user name and email
- Better show emoji
- Fix schemes

### CODE REVIEW

File: init.rb [hunk 1]
```diff
@@ -11,30 +11,29 @@ def prompt(message, default)
...
 
-default_package_name = 'MyPackage'
+default_package_name = File.basename(Dir.getwd)
 default_bundle_domain = 'no.hyper'
-default_author_name = 'Hyper Interaktiv AS'
-default_author_email = 'ios@hyper.no'
-default_username = 'hyperoslo'
+default_author_name = `git config user.name`.strip
+default_author_email = `git config user.email`.strip
+default_username = default_author_email.split('@').first
```
Comment (zenangst) [inline init.rb:18]: If I use this now then it will use my personal email and show me as the author :/

File: init.rb [hunk 2]
```diff
@@ -11,30 +11,29 @@ def prompt(message, default)
...
 
-file_names.push("#{folder_path}/.slather.yml")
-file_names.push("#{folder_path}/.travis.yml")
+file_names.push("#{folder_path}/circle.yml")
 
 file_names.push("#{folder_path}/SwiftPackage.xcodeproj/project.pbxproj")
 file_names.push("#{folder_path}/SwiftPackage.xcodeproj/project.xcworkspace/contents.xcworkspacedata")
 file_names.push("#{folder_path}/SwiftPackage.xcodeproj/xcshareddata/xcschemes/SwiftPackage-iOS.xcscheme")
-file_names.push("#{folder_path}/SwiftPackage.xcodeproj/xcshareddata/xcschemes/SwiftPackage-Mac.xcscheme")
+file_names.push("#{folder_path}/SwiftPackage.xcodeproj/xcshareddata/xcschemes/SwiftPackage-macOS.xcscheme")
```
Comment (zenangst) [inline init.rb:36]: nice!

File: init.rb [hunk 3]
```diff
@@ -11,30 +11,29 @@ def prompt(message, default)
...
+author_email = prompt('📧  E-mail', default_author_email) || default_author_email
+username = prompt('👑  Username', default_username) || default_username
+use_storyboards = prompt('🤖  Example with Storyboards? - yes/no', default_storyboards) || default_storyboards
 storyboards_example = use_storyboards.downcase == "yes"
 
 file_names = Dir["#{folder_path}/**/*.*"]
 
-file_names.push("#{folder_path}/.slather.yml")
-file_names.push("#{folder_path}/.travis.yml")
+file_names.push("#{folder_path}/circle.yml")
```
Comment (zenangst) [inline init.rb:31]: Oh :)

File: init.rb (hunk 1 above)
Comment (zenangst) [reply init.rb:18]: Would it be better to use an env variable here?

Comment (onmyway133) [reply init.rb:18]: @zenangst I recognise some others are using this too. Just want to make it a bit. Or we could fetch these information from `cocoapods trunk me`?

Comment (zenangst) [reply init.rb:18]: Yeah I totally get that :) I'm just curious how we could fix this for everyone :)

**Review by zenangst — APPROVED** [review]

**Review by vadymmarkov — APPROVED** [review]

**Comment by vadymmarkov** [comment]
@onmyway133 You're our Ruby guru from now on 😉

**Comment by onmyway133** [comment]
@vadymmarkov at least, it's better than Apple Script 😬

**Pull request merged**